Images_And_Graphics_style.css_v1

```diff
@@ -17,29 +17,36 @@ h1 {
 
 h2 {
   color: #000000;
-  font-size: 30px;
+  font-size: 20px;
   text-align: center;
 }
 
 h3 {
   color: #FFF;
-  font-size: 45px;
+  font-size: 26px;
   text-align: center;
-  margin-top: 30px;
+  margin-top: 20px;

```

Comment:
I changed margin-top from 30px to 20px because the heading was at the top more then I wanted it to be.